build-system/tasks/e2e/build-bento-e2e.js

```diff
@@ -47,7 +47,9 @@ function buildEndToEndBinaries(options, bentoComponentName, testFor) {
           wrapper: '',
           remap: {
             'preact': 'react',
-            '.*/preact/compat': 'react',
+            'preact/compat': 'react',
```

Comment:
I recall I needed to do this because some files imported directly from `preact/compat` I assume you must have changed it to make this work.

Reply:
yes. 

before:
* all components import from `#preact` which imports from `preact`;

after:
* standalone/preact:
  * components import from `#preact` which imports from `#preact/internal` which imports from `preact/compat`;
(the standalone binaries bundle preact while the preact binaries don't)
* react:
  * components import from `#preact` which imports from `#preact/external` which imports from `react`;